news/2019-03-31-beatmap-spotlights-winter-2018.md

```diff
@@ -0,0 +1,308 @@
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/taiko/impulse.jpg)](https://osu.ppy.sh/beatmapsets/751401)
+
+There are some mappers that don't aim a lot of their content to the ranked section, but when they do, they never cease to amaze us with fresh innovative content. Of course one of them is [Fallmorph](https://osu.ppy.sh/users/5720389)! The mapper who amazed us with Enantiomorphs three seasons ago, has come back with a new mind-blowing Marathon map, making an amazing mapping rendition to Impulse, a futuristic sounding drumstep track by osu!'s Featured Artist, Culprate.
+
+This map is absolutely worth of being called official content, since it includes high effort mapping skills such as amazing control of contrast between sections, climatic tension, advanced slider velocity changes and hitsounding, the latter being absent of most osu!taiko beatmaps.
+
+Impulse is a dreamy journey through a futuristic utopia, with calm, high spaced notes falling into interesting drum beats, playing a lot with rhythm variance and clever usage of drumrolls, beatifully executed along with synth hitsounded big note ornaments.
+
+1/3? 1/4? 1/6? 1/8? This map features them all! As we go deeper into this atmosphere of textures, we find ourselves falling progressively into a more difficult, technical style of mapping: The track goes on, but it doesn't get any boring, it just gets better and better...
+
+Saying any more about this map could be considered an spoiler, so how about you give it a try? Trust us, it won't dissapoint you!
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/taiko/epimerization.jpg)](https://osu.ppy.sh/beatmapsets/754062)
+
+This beatmap by [Alchyr](https://osu.ppy.sh/users/4993032) is completed as a single story. It begins with the first silent scene following the quiet melody. after that, the story changes dramatically, just like a powerful storm of finishers and spinners. The visual difficulty of finishers further enhances it. Then the storm turns into something like 1/6 arrows, and we can easily experience how difficult handswaping is. 
+
+After a long break, you can see a very impressive part making use of tremendous density changes. In the end, the story finally finishes with the perfect part summarizing all its concepts. The great attractive thing about this story is that it is not boring but inversely enjoyable despite repeating the same pattern, of course because of the perfection of mapping.
+
+まさしく，一つの物語として成す譜面です．まず，物語は静寂たる音色とそれに合った静けさに始まります．束の間，まるで大音符とスピナーの嵐かのように劇的に物語は変化します．大音符の視認性は拍車をかかけます．そして嵐は1/6の矢へと移り変わり，逆手の極みを篤と見せつけてくるのです．長い静寂を挟み，強烈な密度変化を取り入れた，凄まじく印象的な場面が広がると，最後はこれまでのすべてのコンセプトを合わせた場面によって物語は終焉を迎えます．同じ配置を繰り返しているようで，つまらぬどころか面白い，それはもちろん，譜面制作の極致に至るこの譜面の持つ趣のまさに一つと言えましょう．
```

Comment:
can some context be given as to why there's a Japanese paragraph

Reply:
Toorun asked me to include it as well. It is the Japanese version of the comment. Personally, I see no harm in including it.